test/resources/fonts/web-fonts/empty.html

```diff
@@ -0,0 +1 @@
+<div style="font-family: Font1"></div>
```

Comment:
Must use the font which will be tested in: https://github.com/bubkoo/html-to-image/blob/128dc3edfde95d6ac636f2756630f5cbd6f7c8df/test/spec/options.spec.ts#L141-L152